src/crypto/symmetric_key.h

```diff
@@ -19,7 +19,7 @@ namespace crypto
     uint8_t iv[SIZE_IV] = {};
 
     // 12 bytes IV with 8 LSB are unique sequence number
-    // and 4 MSB are 4 LSB of unique id (Node id or View)
+    // and 4 MSB are 4 LSB of term (with last bit indicating a snapshot)
```

Comment:
Can we use "view" rather than "term" here, since this is a cross-consensus concern.

Reply:
As discussed, let's stick to "term" as this comes from the KV directly, which refers to consensus view as "term" so that it's not confused with a `kv::Tx`'s view.